.stylelintrc.js

```diff
@@ -6,8 +6,17 @@
  */
 
 module.exports = {
+  extends: ['stylelint-config-recommended', 'stylelint-config-prettier'],
   plugins: ['stylelint-copyright'],
   rules: {
     'docusaurus/copyright-header': true,
+    'selector-pseudo-class-no-unknown': [
+      true,
+      {
+        ignorePseudoClasses: ['global'],
+      },
+    ],
+    'no-descending-specificity': null,
+    'declaration-block-no-duplicate-custom-properties': null,
```

Comment:
These are probably fixable, but wasn't confident about which ones to delete
```
packages/docusaurus-cssnano-preset/src/remove-overridden-custom-properties/__tests__/fixtures/important_rule.css
 4:3  ✖  Unexpected duplicate "--color-primary"    declaration-block-no-duplicate-custom-properties
 6:3  ✖  Unexpected duplicate "--color-header"     declaration-block-no-duplicate-custom-properties
 7:3  ✖  Unexpected duplicate "--color-secondary"  declaration-block-no-duplicate-custom-properties

packages/docusaurus-cssnano-preset/src/remove-overridden-custom-properties/__tests__/fixtures/normal.css
 3:3  ✖  Unexpected duplicate "--color-primary"  declaration-block-no-duplicate-custom-properties
 5:3  ✖  Unexpected duplicate "--color-primary"  declaration-block-no-duplicate-custom-properties
```

Reply:
Those two files were testing duplicate properties, because we have a CSS minifier that removes these duplicates

Reply:
Right, didn't click that it was in a test file. Probably better to override for those files instead of turning off the rule entirely